backend/spellbook/management/commands/export_variants.py

```diff
@@ -58,6 +58,8 @@ def handle(self, *args, **options):
                 'variants': [prepare_variant(v) for v in variants_source],
             }
 
+            result = humps.camelize(result)
```

Comment:
Why camelization? What does it do?

Reply:
Makes the result camel case in preparation for the client to use.